libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultApolloExtension.kt

```diff
@@ -160,6 +160,9 @@ abstract class DefaultApolloExtension(
     project.tasks.register(ModelNames.downloadApolloSchema(), ApolloDownloadSchemaTask::class.java) { task ->
       task.group = TASK_GROUP
       task.projectRootDir = project.rootDir.absolutePath
+      task.doLast {
+        it.logger.lifecycle("Apollo: using './gradlew downloadApolloSchema' is deprecated. Please use the Apollo Kotlin cli for one-time downloads or the introspection {} block for Gradle downloads. See https://go.apollo.dev/ak-download-schema.")
```

Comment:
The go link needs to be updated to point to https://www.apollographql.com/docs/kotlin/advanced/plugin-recipes#downloading-a-schema right?

Reply:
Good catch 👍 https://github.com/apollographql/shortlinks/commit/d24a659f82f822689a70c5c363948032c3fcdf40